benchmark/build.gradle.kts

```diff
@@ -18,8 +18,10 @@ dependencies {
 }
 
 jmh {
-    warmupIterations.set(1)
-    iterations.set(5)
-    fork.set(2)
-    benchmarkMode.set(listOf("ss"))
+    warmupIterations = 1
+    iterations = 5
+    fork = 2
+    benchmarkMode = listOf("ss")
+    includes = listOf("SingleThreadedBenchmark")
```

Comment:
this is run more often than the multithreaded - which can be enabled by commenting this out